portfolio/src/main/webapp/blog.html

```diff
@@ -143,10 +143,15 @@ <h5>Snow Sun</h5>
                 <span class="sr-only">Next</span>
             </a>
         </div>
+    </div>
+    <div id = "content6">
         <div id="content">
             <p>Click here to get a random Ethiopian proverb:</p>
             <button onclick="addRandomGreeting()">Proverb</button>
             <div id="greeting-container"></div>
+             <p>More proverbs</p>
```

Comment:
Indent to the same level as the surrounding lines.

Reply:
Fixed!